From: Jordi Mallach <jordi@debian.org>
Description: Remove an unknown "LO" section from the manpage to avoid
             man errors.
Index: alsa-utils-1.0.21/seq/aseqnet/aseqnet.1
===================================================================
--- alsa-utils-1.0.21.orig/seq/aseqnet/aseqnet.1	2009-08-31 17:13:36.000000000 +0200
+++ alsa-utils-1.0.21/seq/aseqnet/aseqnet.1	2009-09-04 22:19:32.000000000 +0200
@@ -1,5 +1,4 @@
 .TH aseqnet 1 "January 1, 2000"
-.LO 1
 .SH NAME
 aseqnet \- ALSA sequencer connectors over network
 
